### DISCUSSION TITLE
절대경로 alias 오류

### DISCUSSION THREAD

**Discussion by baechanghyeon** [WantedPreonboardingFE5team/pre-onboarding-7th-2-1-5#38] [General]
<img width="1039" alt="스크린샷 2022-11-09 오후 3 14 47" src="https://user-images.githubusercontent.com/28972561/200753599-f9193b09-1765-45b6-b5b3-1a43e346899c.png">

혹시 절대경로 관련해서 이런 오류 뜨시는 분 있나요?

alias 한 경로 값을 못 읽어서 저런 에러가 나는거 같습니다.

**Comment by jaeyeong815** [comment]
저 에러였는지 기억이 잘 안나는데 절대경로 에러 뜨길래 일단 ../ 로 고쳐서 하고있었습니다..!

**Reply by baechanghyeon** [reply]
../ 이라는게 alias 설정 전에 기본 경로 말씀하신거죠 ??

**Reply by jaeyeong815** [reply]
네넹 기존 쓰던 방식이요 ~,~!!

**Reply by hyoniiii** [reply]
저도 같은 에러 나서 기존 방식으로 사용하고 있어요

**Reply by baechanghyeon** [reply]
앗.. 네 알겠습니다 !
